packages/buckram/assets/styles/components/specials/_contributors.scss

```diff
@@ -6,75 +6,75 @@
 //                           Contributors                                     //
 //============================================================================//
 
-div.contributors {
-  border-top: 1px solid;
-  margin-top: 2rem;
-
-  &.page {
-    border-top: 0;
+.contributors {
+  .chapter-authors, .book-contributors {
+    margin-top: $contributors-margin-top;
+    font-family: $body-font-family;
+  }
 
-    h3 {
-      font-family: $font-1;
-      font-size: $h1-font-size;
-    }
+  .about-authors {
+    margin-top: $contributors-margin-top;
+    margin-bottom: $contributors-margin-bottom;
+    text-align: $contributors-text-align;
+  }
 
-    .contributor_name {
-      font-size: 25px;
-    }
+  .contributor__type {
+    margin-top: $contributors-margin-top;
+    margin-bottom: $contributors-margin-bottom;
   }
 
-  .contributor_name_and_links {
-    margin-top: 2rem;
+  .contributor__name__and__links {
+    margin-top: $contributors-margin-top;
   }
 
-  .about-authors {
-   margin: 2rem 0;
-   text-align: left;
+  // TODO: replace clearfix hack with better CSS Grid solution when Prince supports grid layout
+  .contributor__name__and__links:after {
+    content: "";
+    display: table;
+    clear: both;
   }
 
-  .contributor_profile_picture {
-    float: left;
-    @if $type == 'epub' {
-      height: 220px;
-      width: 220px;
-    }
-    max-width: 155px;
-    max-height: 155px;
-    border-radius: 50%;
-    margin-right: 1.5rem;
+  .contributor__profile__picture {
+    float: $contributor-profile-picture-float;
+    margin-top: $contributor-profile-picture-margin-top;
+    margin-right: $contributor-profile-picture-margin-right;
+    margin-bottom: $contributor-profile-picture-margin-bottom;
+    margin-left: $contributor-profile-picture-margin-left;
+    width: 33%;
```

Comment:
Not sure about best way to handle image width elegantly for web and export routines. Image we're working with will always be 400x400 squares.